soundfile.py

```diff
@@ -968,6 +990,120 @@ def read(self, frames=-1, dtype='float64', always_2d=False,
                 out[frames:] = fill_value
         return out
 
+    def read_raw(self, frames=-1, dtype=None):
```

Comment:
That was my initial thought too but the raw_read/write functions actually require that you specify a number of bytes that is a multiple of the audio file frame size. From the libsndfile docs:

> The number of bytes read or written must always be an integer multiple of the number of channels multiplied by the number of bytes required to represent one sample from one channel.

Hence, it made more sense to make the interface this way.